src/frontend/src/handler/mod.rs

```diff
@@ -24,13 +24,13 @@ pub mod create_mv;
 pub mod create_source;
 pub mod create_table;
 mod describe;
+pub mod dml;
 pub mod drop_mv;
 pub mod drop_table;
 mod explain;
 mod flush;
 #[allow(dead_code)]
```

Comment:
remove allow dead code?

Reply:
I tried but seems we still have some missing part. Let's remove them later.